plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/FunnyCommandsConfiguration.java

```diff
@@ -111,7 +112,8 @@ public static FunnyCommands createFunnyCommands(FunnyGuilds plugin) {
                 .command("top", commands.top, TopCommand.class)
                 .command("validity", commands.validity, ValidityCommand.class)
                 .command("war", commands.war, WarCommand.class)
-                .command("tnt", commands.tnt, TntCommand.class);
+                .command("tnt", commands.tnt, TntCommand.class)
+                .command("pp", commands.helpRequest, HelpRequestCommand.class);
```

Comment:
`pp` to tylko skrótowy alias, zmień na `helprequest` żeby było to zgodnie tak jak w innych komendach